onnxruntime/core/framework/insert_cast_transformer.h

```diff
@@ -3,7 +3,7 @@
 
 #pragma once
 #include "core/graph/graph_viewer.h"
-#include "core/graph/graph_transformer.h"
+#include "core/optimizer/graph_transformer.h"
```

Comment:
If optimizer depends on framework, then framework should not include any header file in optimizer.

Reply:
done!